packages/esm-patient-conditions-app/src/conditions/conditions-form.component.tsx

```diff
@@ -53,12 +60,18 @@ type ViewState = IdleState | SearchState | ConditionState | SubmitState;
 const ConditionsForm: React.FC<ConditionsFormProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
   const session = useSessionUser();
+  const layout = useLayoutType();
   const [clinicalStatus, setClinicalStatus] = React.useState('active');
   const [endDate, setEndDate] = React.useState(null);
   const [onsetDate, setOnsetDate] = React.useState(null);
   const [viewState, setViewState] = React.useState<ViewState>({
     type: StateTypes.IDLE,
   });
+  const [lightMode, setLightMode] = useState<boolean>();
```

Comment:
I think this is rather a `useMemo`.

```
useMemo(() => layout !== 'desktop', [layout])
```

No need for the whole ceremony (and its a perf killer, too).